flashinfer/moe_ep/kernel_src/next_cutedsl_megamoe/src/sources/helpers/utils.py

```
@@ -0,0 +1,100 @@
+
+def validate_static_integer_tuple(value: Tuple, *, field_name: str) -> None:
+    for element in value:
+        if isinstance(element, tuple):
+            validate_static_integer_tuple(element, field_name=field_name)
+        elif not isinstance(element, int):
+            raise TypeError(f"{field_name} must contain Python ints, got {type(element)}.")
+
+
+def flatten_shape_stride(shape: Tuple, stride: Tuple) -> List[Tuple[IntegerType, IntegerType]]:
+    pairs: List[Tuple[IntegerType, IntegerType]] = []
+    for size, step in zip(shape, stride):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)